src/DisplayWindow.cpp

```diff
@@ -8,6 +8,8 @@
 #include "PluginAPI.h"
 #include "FrameBuffer.h"
 
+#include <Graphics/OpenGLContext/ThreadedOpenGl/opengl_Wrapper.h>
+
```

Comment:
Top level code must not use GL code directly, never.
This code does not know about graphics backend. It can be Vulcan or DX backends, top level code should work the same. All access to graphics must be done via graphics::Context from src\Graphics\Context.h.
You need to extend it with a new method. Check getMaxLineWidth() as an example. You may add one specialized method, like `getMaxMSAALevel()` or write a general one like
`int getIntegerParameter(Parameter _name);`
and add new `SpecialParameterClass` in Parameters.h with its proper implementation for GL_MAX_SAMPLES parameter. The second option needs more efforts and it is not evident yet that this general function is necessary.

Reply:
this should be fixed, I hope I put the `getMaxMSAALevel()` in the right place now

Reply:
Thanks! I'll check it a bit later. I need to finish with #2405